### DISCUSSION TITLE
Please stop it with the 2FA virtual support and allow real support.  Your process is overly complicated.

### DISCUSSION THREAD

**Discussion by kenlnetherland** [community/community#121866] [Code Security]
### Select Topic Area

Product Feedback

### Body

I spent 3 hours trying to get 2FA to work.  I had GitHub on my authenticator app already and then you force me to use 2FA on my new account after losing all my intellectual property from before.  3 hours guys!  I am well aware you are not moving away from this, but please improve the process.  Granted, the MS Auth app is not particularly user friendly, but if you are going to dictate using this, do better!

Steps I took on Android;
- Clicked on the 2FA popup
- Scanned the QR code
- Pulled up the Authenticator app with no indication of what to do next.  ?????
- After dumbfounded for a long time, determined that I have to remove the old Github account, click the + sign and scan from there.

Don't send me documentation, improve your product!

**Comment by leofun01** [comment]
It's wrong request. We should require to make 2FA optional.
